seed-job/Jenkinsfile

```diff
@@ -38,7 +38,7 @@ pipeline {
     }
         stage('Parse index') {
       steps {
-          sh "python seed-job/parser.py"
+          sh "python seed-job/parser.py ${JENKINSFILE_GIT_BRANCH} ${REGISTRY_URL}"
```

Comment:
this registry url is with port?

Reply:
Yes. Example `REGISTRY_URL` could be `192.168.122.182:5000`. I have ensured to cover that in the README.